### PR TITLE
Update API specification for various party details

### PR DESCRIPTION
**Pull request by jguertin**
Reorganize party specifications to support individual and business names, along with the addition of a shared address model

### CODE REVIEW

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 1]
```diff
@@ -480,19 +540,33 @@ components:
...
-      title: individual-name
+    address:
+      title: address
       type: object
       properties:
-        first:
+        street:
           type: string
-        last:
+        sreetAdditional:
```
Comment (WalterMoar) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml]: Small typo here.

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 2]
```diff
@@ -233,36 +238,47 @@ paths:
...
-                      - code: string
-                        name: string
-                        address: string
-                        city: string
-                        province: string
-                        postalCode: string
+                      - businessName: Spacely Sprockets
+                        address:
+                          street: 123 Fake St
+                          sreetAdditional: Suite 501
```
Comment (WalterMoar) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml]: Typo here.

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 3]
```diff
@@ -279,18 +295,42 @@ paths:
...
+                      street: 742 Evergreen Terrace
+                      city: Springfield
+                      region: BC
+                      country: CA
+                      postalCode: A1A 1A1
+                  securedParties:
+                    - businessName: Spacely Sprockets
+                      address:
+                        street: 123 Fake St
+                        sreetAdditional: Suite 501
```
Comment (WalterMoar) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml]: Likewise.

**Review by WalterMoar — CHANGES_REQUESTED** [review]
Some typos with `sreetAdditional` to be fixed.

File: ppr-api/docs/api_contract/ppr-api.v1.yaml (hunk 3 above)
Comment (jguertin) [reply ppr-api/docs/api_contract/ppr-api.v1.yaml]: Nice catch

**Review by WalterMoar — APPROVED** [review]

**Review by bryan-gilbert — APPROVED** [review]

**Pull request merged**